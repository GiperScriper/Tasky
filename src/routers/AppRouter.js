import React from 'react';
import { BrowserRouter, Route, Switch } from 'react-router-dom';
import PrivateRoute from './PrivateRoute';

import Login from 'components/Login';
import Layout from 'components/Layout';

const appRouter = () => (
  <BrowserRouter>
    <Switch>
      <Route path="/login" component={Login} />
      <PrivateRoute path="/" component={Layout} />
    </Switch>
  </BrowserRouter>
);

export default appRouter;
