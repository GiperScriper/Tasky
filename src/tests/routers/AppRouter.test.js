import React from 'react';
import { shallow } from 'enzyme';
import AppRouter from '@/routers/AppRouter';

describe('<AppRouter />', () => {
  it('should render AppRouter component correctly', () => {
    const wrapper = shallow(<AppRouter />);
    expect(wrapper).toMatchSnapshot();
  });
});
