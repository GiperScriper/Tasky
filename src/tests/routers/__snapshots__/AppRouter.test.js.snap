// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppRouter /> should render AppRouter component correctly 1`] = `
<React.Fragment>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/projects"
  />
  <Route
    component={[Function]}
    path="/projects/:id"
  />
</React.Fragment>
`;
